doc/nrf/ug_thingy91.rst

```diff
@@ -161,49 +161,64 @@ You can choose the method based on the availability or absence of an external de
    In a final product, you must use your own, secret keys.
    See :ref:`ug_fw_update_development_keys` for more information.
 
+.. _build_pgm_vsc:
+
 Building and programming using VS Code extension
 ================================================
 
 |vsc_extension_instructions|
 
-.. _build_pgm_segger:
+Complete the following steps to build |NCS| projects with |VSC|:
 
-Building and programming using SEGGER Embedded Studio
-=====================================================
+1. Start |VSC|.
 
-.. include:: gs_programming.rst
-   :start-after: build_SES_projimport_open_start
-   :end-before: build_SES_projimport_open_end
+   If you installed the |NCS| using the :ref:`gs_app_tcm`, you can click the :guilabel:`Open VS Code` button next to version you installed.
 
-..
+#. Click :guilabel:`Create new application from sample` from the :guilabel:`Welcome` panel.
 
-   .. figure:: images/ses_thingy_configuration.png
-      :alt: Opening the Asset tracker v2 application
+   The :guilabel:`New Application` screen opens.
 
-      Opening the Asset tracker v2 application for the thingy91_nrf9160_ns build target
+#. Provide the following information to create the application:
 
-   .. note::
+   * For :guilabel:`Application type`, select :guilabel:`Freestanding`.
```

Comment:
Just a doubt that wouldn't the customers be using the existing applications/samples (from "add an existing applications") that are there in the NCS to build the program, or do they create their applications?
In Segger, they used to select the existing application/sample from the ncs.

![image](https://user-images.githubusercontent.com/91891495/168963672-ea5ae0c5-2fe2-4e8c-93cc-6a8d8ccd637b.png)

Reply:
**Create a new application from sample** is what the [VS Code documentation](https://nrfconnect.github.io/vscode-nrf-connect/connect/create_app.html) uses. It also means you get to use the drop-down or **Browse samples** instead of having to go through the folder structure.